justplay/src/main/java/inaka/justplay/core/JustPlay.java

```diff
@@ -8,17 +8,21 @@
 
 import java.io.IOException;
 
+import inaka.justplay.R;
+
 
 /**
- * Created by emi on 1/8/16.
+ * Created by tenmaster on 1/8/16.
```

Comment:
According to our Android internal conventions, we prefer not to add headers indicating the creator/owner of the file